client/blocks/reader-full-post/style.scss

```diff
@@ -412,6 +412,24 @@
 	margin: 28px 0 26px;
 }
 
+.reader-full-post .embed-youtube,
+.reader-full-post .embed-vimeo {
+	display: block;
+	position: relative;
+	padding: 25px 0 56.25%;
+
+	iframe {
+		height: 100%;
```

Comment:
Yes, without setting a height to the container to maintain its aspect ratio. But I think JS would be probably be a more reasonable way to go about this to be make sure other embeds are covered.